snap

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+(cd ha-core ; cp ../tests/components/plugwise/test_setup.py tests/components/plugwise/ ; source venv/bin/activate; pytest tests/components/plugwise/test_setup.py ; deactivate; cd ..)
```

Comment:
snap is just for very quickly testing locally - committed to the branch to potentially share workload